lib/renderer/base_chart_painter.dart

```diff
@@ -93,14 +93,16 @@ abstract class BaseChartPainter extends CustomPainter {
     canvas.scale(1, 1);
     drawBg(canvas, size);
     drawGrid(canvas);
-    if (datas != null && datas!.isNotEmpty) {
-      drawChart(canvas, size);
-      drawRightText(canvas);
-      drawDate(canvas, size);
-      if (isLongPress == true) drawCrossLineText(canvas, size);
-      drawText(canvas, datas!.last, 5);
-      drawMaxAndMin(canvas);
-      drawNowPrice(canvas);
+    if (datas != null) {
+      if (datas!.isNotEmpty) {
```

Comment:
Why do this? I think ` if (datas != null && datas!.isNotEmpty)` is better.

Reply:
i think it would avoid unexpected error. i'll change it back